src/apps/core/app.js

```diff
@@ -651,9 +655,15 @@ define(function(require) {
 			});
 		},
 
-		onRequestStart: function($spinner) {
+		onRequestStart: function(args) {
 			var self = this,
-				waitTime = 250;
+				waitTime = 250,
+				$spinner = args.spinner;
+
+			// If indicated, bypass progress indicator display/hide process
+			if (_.get(args, 'bypassProgressIndicator', false)) {
```

Comment:
You should unpack `args` parameters in their own variable. It makes the code easier to read and also forces you to set default values at the same time/location (on variable declaration).

```suggestion
			if (isBypassingProgressIndicator) {
```